features/networking/ovn_ipsec.feature

```diff
@@ -0,0 +1,158 @@
+       | ["metadata"]["namespace"]                                  | <%= project.name %>                                                                |
+    #Below socat command will send ESP traffic at a length less than 40
+       | ["spec"]["containers"][0]["command"]                       | ["bash", "-c", "socat /dev/random ip-sendto:<%= cb.host_ip %>:<%= cb.protocol %>"] |
+       | ["spec"]["containers"][0]["securityContext"]["privileged"] | true                                                                               |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+       | name=network-pod |
+    #Hostnetwork pod for worker1 to leverage tcpdump tool. Has to be run in privileged mode
+    Given I obtain test data file "networking/net_admin_cap_pod.yaml"
+    When I run oc create as admin over "net_admin_cap_pod.yaml" replacing paths:
+       | ["spec"]["nodeName"]                                       | <%= cb.workers[1].name %> |
+       | ["metadata"]["namespace"]                                  | <%= project.name %>       |
+       | ["metadata"]["name"]                                       | hostnw-pod-worker1        |
+       | ["spec"]["containers"][0]["securityContext"]["privileged"] | true                      |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+       | name=network-pod |
+    And evaluation of `pod.name` is stored in the :hello_pod_worker1 clipboard
+    #Make sure you got some packets captured at the receiver node.The socat we used earlier will dump any ESP packets less that 40 in length (these are invalid in our clusters)
+    When admin executes on the "<%= cb.hello_pod_worker1 %>" pod:
+       | bash | -c | timeout  --preserve-status 60 tcpdump -c 2 -i br-ex "esp and less 40" |
+    Then the step should succeed 
+    And the output should not contain "0 packets captured"
+  
+  # @author anusaxen@redhat.com
+  # @case_id OCP-38845
+  @admin
+  @destructive
+  Scenario: Segfault on pluto IKE daemon should result in restarting pluto daemon and corresponding ovn-ipsec pod
+    Given the env is using "OVNKubernetes" networkType
+    And the IPSec is enabled on the cluster
+    Given I store all worker nodes to the :workers clipboard
+    #Getting ovn-ipsec pod name for a corresponding worker node in sight
+    When I run the :get admin command with:
+      | resource      | pod                                   |
+      | l             | app=ovn-ipsec    |
+      | fieldSelector | spec.nodeName=<%= cb.workers[0].name %> |
+      | n             | openshift-ovn-kubernetes              |
+      | o             | jsonpath={.items[*].metadata.name}    |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :ovn_ipsec_pod clipboard
+    Given I use the "<%= cb.workers[0].name %>" node
+    #Simulating segfault on pluto IKE dameon
+    And I run commands on the host:
+      | pgrep -SEGV pluto|
```

Comment:
Should be "pgrep pluto \| pkill -SEGV pluto"?

Reply:
Yea supposed to be ```pkill -SEGV pluto``` as discussed above. My latest commit made it strange and another commit was on the way to re-fix it.done

Reply:
Of course we can try lot of variations here